win32ss/user/user32/windows/window.c

```diff
@@ -283,6 +283,17 @@ User32CreateWindowEx(DWORD dwExStyle,
         plstrClassVersion = &lstrClassVersion;
     }
 
+    /* Check process default layout */
+    if (!hWndParent)
```

Comment:
As done on Windows, setting the LAYOUTRTL flag should be done in the kernel.
After analysis, a good place would be in co_UserCreateWindowEx() or, better, in IntCreateWindow().